tools/indexer/example/src/configs.rs

```diff
@@ -46,10 +46,16 @@ pub(crate) struct InitConfigArgs {
     pub genesis: Option<String>,
     #[clap(short, long)]
     /// Download the verified NEAR genesis file automatically.
-    pub download: bool,
+    pub should_download_genesis: bool,
     /// Specify a custom download URL for the genesis-file.
     #[clap(long)]
     pub download_genesis_url: Option<String>,
+    #[clap(short, long)]
```

Comment:
I think it does not make sense to have `short` versions of `should-download-genesis` commands